src/libsync/theme.cpp

```diff
@@ -81,9 +84,6 @@ QString Theme::statusHeaderText(SyncResult::Status status) const
     case SyncResult::SetupError:
         resultStr = QCoreApplication::translate("theme", "Setup Error");
         break;
-    case SyncResult::SyncPrepare:
-        resultStr = QCoreApplication::translate("theme", "Preparing to sync");
```

Comment:
I think the idea is that user complains "Why is the sync running when there should be nothing to sync".
Maybe that is no longer relevant because of other changes of the UI.  But probablt it still deserve a different message.